lib/formulator.ex

```diff
@@ -2,31 +2,86 @@ defmodule Formulator do
   use Phoenix.HTML
   alias Formulator.HtmlError
 
+  @doc  """
+  Returns an html input with an associated label.
+  If there are errors associated with the field, it will also output a span
+  tag with the errors.
+
+  ## Options
+
+    * `:label` - when false, does not create a label tag. Instead, an `aria-label`
```

Comment:
Would you be against using `label` for both of these? I realize overloading that may be a little more annoying for us as maintainers, but I like it better as a consumer of the library. Mainly just because`label_text` is more verbose than `label` and you should never be using both of these at the same time anyway. It's also what I would expect coming from simple_form (not that thats super important).

I don't have strong feelings about this, so up to you.